patches/server/0112-Fix-chest-visual-open-bug.patch

```diff
@@ -0,0 +1,44 @@
+From 5a115d5d06dfb6ade6dbcdc92c6b13c508e8dcd2 Mon Sep 17 00:00:00 2001
+From: FabrisNick <niccolofabris05@gmail.com>
+Date: Sun, 25 May 2025 20:37:52 +0200
+Subject: [PATCH] Fix chest visual open bug
+
+---
+ .../com/hpfxd/pandaspigot/config/PandaSpigotConfig.java    | 4 ++++
+ src/main/java/net/minecraft/server/PlayerConnection.java   | 7 +++++++
+ 2 files changed, 11 insertions(+)
+
+diff --git a/src/main/java/com/hpfxd/pandaspigot/config/PandaSpigotConfig.java b/src/main/java/com/hpfxd/pandaspigot/config/PandaSpigotConfig.java
+index 1d8e4d105..47137cd6e 100644
+--- a/src/main/java/com/hpfxd/pandaspigot/config/PandaSpigotConfig.java
++++ b/src/main/java/com/hpfxd/pandaspigot/config/PandaSpigotConfig.java
+@@ -123,4 +123,8 @@ public class PandaSpigotConfig {
+     @Comment("Whether player IP addresses should be logged by the server. This does not impact\n" +
+         "the ability of plugins to log the IP addresses of players.")
+     public boolean logPlayerIpAddresses = true;
++
++    @Comment("Prevents sending interact packets while an inventory is open. This fixes the bug\n" +
++        "of chests showing as open when the aren't.")
++    public boolean dropInteractPacketsInInventory = true;
```

Comment:
Disable this by default, as it is not the vanilla behavior. It would be good to describe this in the comment as well and make it clear that this may cause issues with some plugin (as described in the original patch).